src/applications/gi/sass/gi.scss

```diff
@@ -71,6 +71,9 @@
     z-index: 500;
     color: var(--vads-color-base-darker);
     background: var(--vads-color-white);
+    p {
+      max-width: 100%;
```

Comment:
The `p` used here should probably be a header element but I will leave that to the team to update. In the mean time, I'm just resetting the max-width value so that it doesn't apply the new USWDS paragraph styles which has a max-width.

![Screenshot 2024-06-05 at 11 07 28 AM](https://github.com/department-of-veterans-affairs/vets-website/assets/872479/be869f91-f910-44df-b7cb-1674ed48732c)